packages/bitcore-node/src/modules/ethereum/p2p/p2p.ts

```diff
@@ -1,31 +1,35 @@
 import { EventEmitter } from 'events';
+import * as os from 'os';
 import Web3 from 'web3';
 import { timestamp } from '../../../logger';
 import logger from '../../../logger';
 import { StateStorage } from '../../../models/state';
 import { ChainStateProvider } from '../../../providers/chain-state';
 import { BaseP2PWorker } from '../../../services/p2p';
+import { IEthNetworkConfig } from '../../../types/Config';
 import { valueOrDefault } from '../../../utils/check';
 import { wait } from '../../../utils/wait';
 import { ETHStateProvider } from '../api/csp';
 import { EthBlockModel, EthBlockStorage } from '../models/block';
 import { EthTransactionModel, EthTransactionStorage } from '../models/transaction';
-import { IEthBlock, IEthTransaction, ParityBlock, ParityTransaction } from '../types';
-import { ParityRPC } from './parityRpc';
+import { ErigonBlock, ErigonTransaction, IEthBlock, IEthTransaction } from '../types';
```

Comment:
did you mean to call this erigon?

Reply:
Yes, I segregated the Parity/Erigon unifying to the `gethSupport` (#3407)  PR. I renamed them in there